pySDC/projects/DAE/sweepers/fully_implicit_DAE.py

```diff
@@ -118,11 +107,14 @@ def implSystem(params):
                     System to be solved as implicit function.
                 """
 
-                params_mesh = P.dtype_f(P.init)
-                params_mesh[:] = params
+                if type(params) == DAEMesh:
```

Comment:
I think we also don't need this anymore...? Maybe we need to do sth like `params_mesh[...] = params`?